scipy/linalg/tests/test_lapack.py

```diff
@@ -3299,95 +3299,104 @@ def test_gges_tgsen(dtype):
     atol = 1e-4
-    
+
     tgsyl = get_lapack_funcs(('tgsyl'), dtype=dtype)
-    rans,lans,scale,dif,info = tgsyl(a,b,c,d,e,f)
-    
+    rout, lout, scale, dif, info = tgsyl(a, b, c, d, e, f)
+
     assert_equal(info, 0)
-    assert_allclose(scale,1.0,rtol=0,atol=np.finfo(dtype).eps*100,err_msg="SCALE must be 1.0")
-    assert_allclose(dif,0.0,rtol=0,atol=np.finfo(dtype).eps*100,err_msg="DIF must be nearly 0")
-    assert_allclose(rans,r,atol=atol,err_msg='Solution for unknown R of Sylvester equation is not correct')
-    assert_allclose(lans,l,atol=atol,err_msg='Solution for unknown L of Sylvester equation is not correct')
-    
+    assert_allclose(scale, 1.0, rtol=0, atol=np.finfo(dtype).eps*100,
+                    err_msg="SCALE must be 1.0")
+    assert_allclose(dif, 0.0, rtol=0, atol=np.finfo(dtype).eps*100,
+                    err_msg="DIF must be nearly 0")
+    assert_allclose(rout, rans, atol=atol,
+                    err_msg="Solution for R is incorrect")
+    assert_allclose(lout, lans, atol=atol,
+                    err_msg="Solution for L is incorrect")
+
+
 @pytest.mark.parametrize('dtype', REAL_DTYPES)
 @pytest.mark.parametrize('trans', ('N', 'T'))
-@pytest.mark.parametrize('ijob', [0,1,2,3,4])
-def test_tgsyl(dtype,trans,ijob):
-    
+@pytest.mark.parametrize('ijob', [0, 1, 2, 3, 4])
+def test_tgsyl(dtype, trans, ijob):
+
     seed(2023)
     atol = 1e-2
-    if(dtype==np.float32):
+    if (dtype == np.float32):
         atol = 5e-2
-    
+
     m, n = 10, 15
-    
-    a,_ = schur(generate_random_dtype_array([m, m], dtype=dtype))
-    d   = np.triu(generate_random_dtype_array([m, m], dtype=dtype))
-    b,_ = schur(generate_random_dtype_array([n, n], dtype=dtype))
-    e   = np.triu(generate_random_dtype_array([n, n], dtype=dtype))
-    c   = generate_random_dtype_array([m, n], dtype=dtype)
-    f   = generate_random_dtype_array([m, n], dtype=dtype)
 
-    tgsyl = get_lapack_funcs(('tgsyl'), dtype=dtype)
-        
-    r,l,scale,dif,info = tgsyl(a,b,c,d,e,f,trans=trans,ijob=ijob)
-    assert_equal(info,0,err_msg="INFO is non-zero")
+    a, _ = schur(generate_random_dtype_array([m, m], dtype=dtype))
+    d = np.triu(generate_random_dtype_array([m, m], dtype=dtype))
+    b, _ = schur(generate_random_dtype_array([n, n], dtype=dtype))
+    e = np.triu(generate_random_dtype_array([n, n], dtype=dtype))
+    c = generate_random_dtype_array([m, n], dtype=dtype)
+    f = generate_random_dtype_array([m, n], dtype=dtype)
 
-    assert scale>=0.0,"SCALE must be non-negative"
-    if (ijob==0):
-        assert_allclose(dif,0.0,rtol=0,atol=np.finfo(dtype).eps*100,err_msg="DIF must be 0 for ijob =0")
+    tgsyl = get_lapack_funcs(('tgsyl'), dtype=dtype)
+    rout, lout, scale, dif, info = tgsyl(a, b, c, d, e, f,
+                                         trans=trans, ijob=ijob)
+
+    assert_equal(info, 0, err_msg="INFO is non-zero")
+    assert scale >= 0.0, "SCALE must be non-negative"
+    if (ijob == 0):
```

Comment:
Here and below the `if` statements don't require parentheses around them

Reply:
Fixed in latest commit. Thanks!

Reply:
The failing tests seem unrelated to my changes. I just rebased after removing parantheses and pushed.